lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1634,6 +1634,17 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
   };
 
   switch (II->getIntrinsicID()) {
+  case Intrinsic::assume: {
+    // llvm.assume translation is currently supported only within
+    // SPV_INTEL_optimization_hints extension, ignore it otherwise, since it's
+    // an optimization hint
+    if (BM->isAllowedToUseExtension(
+            ExtensionID::SPV_INTEL_optimization_hints)) {
+      SPIRVValue *Condition = transValue(II->getArgOperand(0), BB);
```

Comment:
Do you need to add the CapabilityOptimizationHintsINTEL capability here, also?

Reply:
I'd say, it's not required (the appropriate test has passed), since I've already added:
```
  SPIRVCapVec getRequiredCapability() const override {
    return getVec(CapabilityOptimizationHintsINTEL);
  }
```
in the instruction definition. So I'll remove it for llvm.expect and see what will happen (sorry for debugging throught review, my working machine went down).

Reply:
Okay, LGTM!

(BTW, I sympathize regarding the lockdown: the exact problems are different for everyone, but almost nobody is unaffected by all of this.)